README.md

```
@@ -1,10 +1,11 @@
-# L'Inter des Laurentides (TaCL) Bus for [MonTransit](https://github.com/mtransitapps/mtransit-for-android)
+<!-- DO NOT EDIT: this file is generated by MTREADME.md.MT.sh -->
+# L'Inter des Laurentides (TaCL) buses for [MonTransit](https://github.com/mtransitapps/mtransit-for-android)
 
 <img width="25%" height="25%" src="app-android/src/main/play/listings/en-US/graphics/icon/1.png"/>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)